tests/autoValue.tests.js

```diff
@@ -1,91 +1,106 @@
-import { Meteor } from 'meteor/meteor';
-import expect from 'expect';
-import { Mongo } from 'meteor/mongo';
-import SimpleSchema from 'simpl-schema';
+import { Meteor } from 'meteor/meteor'
+import expect from 'expect'
+import { Mongo } from 'meteor/mongo'
+import SimpleSchema from 'simpl-schema'
+import { callMongoMethod } from './helper'
 
-const collection = new Mongo.Collection('autoValueTestCollection');
+/* global describe, it */
+
+const collection = new Mongo.Collection('autoValueTestCollection')
 const localCollection = new Mongo.Collection('autoValueTestLocalCollection', { connection: null });
 
 [collection, localCollection].forEach((c) => {
   c.attachSchema(new SimpleSchema({
     clientAV: {
       type: SimpleSchema.Integer,
       optional: true,
-      autoValue() {
-        if (Meteor.isServer) return;
-        return (this.value || 0) + 1;
+      autoValue () {
+        if (Meteor.isServer) return
+        return (this.value || 0) + 1
       }
     },
     serverAV: {
       type: SimpleSchema.Integer,
       optional: true,
-      autoValue() {
-        if (Meteor.isClient) return;
-        return (this.value || 0) + 1;
+      autoValue () {
+        if (Meteor.isClient) return
+        return (this.value || 0) + 1
```

Comment:
There're style changes all over the code and it's adding more burden to this big PR already. I'd appreciate if you can revert them.

Reply:
You've done enough work already so I'll be doing this but I thought to also point out.